packages/styles/framework/src/utilities/mixins/_mixins.scss

```
@@ -180,67 +264,22 @@
       margin-right: 0;
     }
 
+    .radio-content {
+      display: flex;
+      flex-direction: column;
+    }
+
     label.#{$type}-label {
       display: block;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)